.github/update-sha.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Script to update the SHA version
+
+# get name and version from dependabots last commit message
+last_commit=$(git log --author=dependabot --pretty=oneline -1)
+name=$(echo $last_commit | grep -o -P '(?<=Bump ).*(?= from)')
+old_version=$(echo $last_commit | grep -o -P '(?<=from ).*(?= to)')
+new_version=$(echo $last_commit | grep -o -P '(?<=to ).*(?= in)')
+
+# find corresponding package file
+cd ../packages/
```

Comment:
The CI executes at the root of the repository, so this relative path won't do what you expect (it'll try to go one level up, to a packages dir, which doesn't exist).

So you have a few options:

1. Change these paths to be relative to the root: `./packages/`, and execute the script from there `.github/update-sha.sh`
2. Change the paths to be relative to the script, you can use `SCRIPT_PATH=$(dirname "$(readlink -f "$0")")` to set a variable to the scripts path, then change this line to `cd $SCRIPT_PATH/../packages/`
3. Eh, maybe something like using `git rev-parse --show-toplevel` to get the full path to the repo's root, and then `cd $(rev-parse --show-toplevel)/packages/`

I think 1 is probably the simplest option, so just do:

```suggestion
cd ./packages/
```